### ISSUE TITLE
[Visual requirement - Logic Apps - Storage]: Luminosity ratio of instruction under "Storage account" edit present on storage tab is less than minimum required ratio 4.5:1.

### ISSUE THREAD

**Issue by takyyon** [Azure/LogicAppsUX#6103]
### Describe the issue

User Experience:
If the color contrast ratio of text is less than the minimum required ratio of 4.5:1, then people with low vision will find it difficult to read the text properly and not able to understand the information which that text conveys and get disoriented. 

Note: User credentials should NOT be included in the bug.

Repro Steps:
- Open https://ms.portal.azure.com/ and login via valid credentials.
- Navigate to the Left Nav bar and select '+New', search 'Logic App' and select it. Press Create button .
- 'Create Logic App' blade appears, enter the details and select "Storage" tab. Navigate all the controls of the selected tab.
- Verify whether luminosity ratio of instruction under "Storage account" edit present on storage tab is less than minimum required ratio 4.5:1 or not.

Actual Result:
Luminosity ratio of instruction under "Storage account" edit present on storage tab is 1.7:1 which is less than minimum required ratio 4.5:1.
Expected Result:

Luminosity ratio of instruction under "Storage account" edit present on storage tab should be equal to or more than minimum required ratio 4.5:1.
Issue: 
Ensures the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (color-contrast -[ https://accessibilityinsights.io/info-examples/web/color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
Target application: 
Create Logic App (Workflow Service Plan) - Microsoft Azure - [https://ms.portal.azure.com/#view/WebsitesExtension/WorkflowAppCreateV3Blade/_provisioningContext~/%7B%22initialValues%22%3A%7B%22subscriptionIds%22%3A%5B%2246841c0e-69c8-4b17-af46-6626ecb15fc2%22%2C%22204898ee-cd13-4332-b9d4-55ca5c25496d%22%2C%223c66da21-607e-49b4-8bdf-f25fbb8f705f%22%5D%2C%22resourceGroupNames%22%3A%5B%5D%2C%22locationNames%22%3A%5B%22eastus%22%2C%22canadacentral%22%2C%22eastus2%22%5D%7D%2C%22telemetryId%22%3A%229059630d-0dcd-42d7-92ba-b98b6ea5b0ba%22%2C%22marketplaceItem%22%3A%7B%22categoryIds%22%3A%5B%5D%2C%22id%22%3A%22Microsoft.Portal%22%2C%22itemDisplayName%22%3A%22NoMarketplace%22%2C%22products%22%3A%5B%5D%2C%22version%22%3A%22%22%2C%22productsWithNoPricing%22%3A%5B%5D%2C%22publisherDisplayName%22%3A%22Microsoft.Portal%22%2C%22deploymentName%22%3A%22NoMarketplace%22%2C%22launchingContext%22%3A%7B%22telemetryId%22%3A%229059630d-0dcd-42d7-92ba-b98b6ea5b0ba%22%2C%22source%22%3A%5B%22LogicAppPlanTypeSelector.ReactView%22%2C%22%7B%20Name%3A%20Part%2C%20Type%3A%20%5B0%5DWebsitesExtension-%5B1%5DLogicAppPlanTypeSelector.ReactView-%5B2%5DAzBladeVirtualLens-%5B5%5DLogicAppPlanTypeSelector.ReactView%2C%20Id%3A%20Part-LogicAppPlanTypeSelector.ReactView-5%20%7D%22%2C%22Part-LogicAppPlanTypeSelector.ReactView-5%22%5D%2C%22galleryItemId%22%3A%22%22%7D%2C%22deploymentTemplateFileUris%22%3A%7B%7D%2C%22uiMetadata%22%3Anull%7D%7D/planType/workflowStandard](https://ms.portal.azure.com/#view/WebsitesExtension/WorkflowAppCreateV3Blade/_provisioningContext~/%7b%22initialValues%22:%7b%22subscriptionIds%22:%5b%2246841c0e-69c8-4b17-af46-6626ecb15fc2%22%2c%22204898ee-cd13-4332-b9d4-55ca5c25496d%22%2c%223c66da21-607e-49b4-8bdf-f25fbb8f705f%22%5d%2c%22resourceGroupNames%22:%5b%5d%2c%22locationNames%22:%5b%22eastus%22%2c%22canadacentral%22%2c%22eastus2%22%5d%7d%2c%22telemetryId%22:%229059630d-0dcd-42d7-92ba-b98b6ea5b0ba%22%2c%22marketplaceItem%22:%7b%22categoryIds%22:%5b%5d%2c%22id%22:%22Microsoft.Portal%22%2c%22itemDisplayName%22:%22NoMarketplace%22%2c%22products%22:%5b%5d%2c%22version%22:%22%22%2c%22productsWithNoPricing%22:%5b%5d%2c%22publisherDisplayName%22:%22Microsoft.Portal%22%2c%22deploymentName%22:%22NoMarketplace%22%2c%22launchingContext%22:%7b%22telemetryId%22:%229059630d-0dcd-42d7-92ba-b98b6ea5b0ba%22%2c%22source%22:%5b%22LogicAppPlanTypeSelector.ReactView%22%2c%22%7b%20Name:%20Part%2c%20Type:%20%5b0%5dWebsitesExtension-%5b1%5dLogicAppPlanTypeSelector.ReactView-%5b2%5dAzBladeVirtualLens-%5b5%5dLogicAppPlanTypeSelector.ReactView%2c%20Id:%20Part-LogicAppPlanTypeSelector.ReactView-5%20%7d%22%2c%22Part-LogicAppPlanTypeSelector.ReactView-5%22%5d%2c%22galleryItemId%22:%22%22%7d%2c%22deploymentTemplateFileUris%22:%7b%7d%2c%22uiMetadata%22:null%7d%7d/planType/workflowStandard)
Element path: 
.msportalfx-text-warning

Snippet: 
<span class="msportalfx-text-warning" data-bind="text: errorMessage">Select a resource group to enable create new storage account option.</span>

How to fix:
Fix any of the following:
  Element has insufficient color contrast of 1.71 (foreground color: #ffba01, background color: #ffffff, font size: 9.8pt (13px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: 
Chrome version 128.0.0.0

### Are you using new designer or old designer

New Designer (With action panels)

### Browser

Chrome version 128.0.0.0

**Comment by hartra344** [comment]
Assigned this one to Shibani to take a look at on ADO
